docs/running-on-yarn.md

```diff
@@ -492,6 +492,13 @@ To use a custom metrics.properties for the application master and executors, upd
   If it is not set then the YARN application ID is used.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.shuffle.service.name</code></td>
+  <td><code>spark_shuffle</code></td>
+  <td>
+    Name of the external shuffle service.
```

Comment:
I think we need more description here. This isn't setting what the service runs as, you have to configure that via yarn, this is what executors use for external shuffle service name when launching the container.

Reply:
That's what "external" word is for, isn't it?
Should I reword it as follows?
> Name of the external shuffle service used by executors.
*The external service is configured and started by YARN (see [Configuring the External Shuffle Service](#configuring-the-external-shuffle-service) for details). Apache Spark distribution uses name `spark_shuffle`, but other distributions (e.g. HDP) might use other names.*

Reply:
many newbie's aren't familiar with what external shuffle service is or even yarn so its best to be clear. How about:

The name of the external shuffle service.
The external shuffle service itself is configured and started by YARN (see Configuring the External Shuffle Service for details). The name specified here must match the name YARN used.